.github/workflows/vast.yaml

```diff
@@ -930,16 +457,17 @@ jobs:
       - name: Build notebooks
         working-directory: examples/notebooks
         run: |
-          export VAST_CONTAINER_REF="${{ needs.determine-version.outputs.vast_container_ref }}"
+          export VAST_CONTAINER_REF="${{ needs.configure.outputs.vast-container-ref }}"
           export VAST_CONTAINER_REGISTRY=ghcr.io
           make docker
           # TODO: An assertion of the actual content of the output would be welcome
           make docker TARGET=clean
 
-  build-nix:
+  vast-nix:
     needs:
-      - determine-version
-    name: Nix Static (${{ matrix.nix.target }})
+      - configure
+    if: ${{ needs.configure.outputs.run-vast-nix == 'true' }}
+    name: VAST (Nix, ${{ matrix.nix.target }})
```

Comment:
Reminder to check for conflicts with #2684 before merging, and maybe leaving a note on that PR to double-check.